### PR TITLE
ATO-344: Add staging performance test stub

### PR DESCRIPTION
**Pull request by CarlyG55**
## What?

Add a stub for performance tests for staging only

### CODE REVIEW

**Comment by CarlyG55** [comment]
This shouldn't be merged until this PR has gone in: https://github.com/govuk-one-login/ipv-stubs-common-infra/pull/167

File: template.yaml [hunk 1]
```diff
@@ -230,6 +236,74 @@ Resources:
...
+          ToPort: 8080
+      VpcId:
+        Fn::ImportValue: !Sub ${VpcStackName}-VpcId
+      Tags:
+        - Key: Name
+          Value: !Sub ${AWS::StackName}-PerfTestContainerServiceSecurityGroup
+        - Key: Service
+          Value: Relying Party Performance Testing stub
+        - Key: Source
+          Value: alphagov/di-auth-stub-relying-party
```
Comment (ethanmills) [inline template.yaml:303]: one login org

Comment (ethanmills) [reply template.yaml:303]: happy for you to change all of these in a new PR

File: template.yaml [hunk 2]
```diff
@@ -672,6 +835,144 @@ Resources:
...
+        - Key: Owner
+          Value: di-orchestration@digital.cabinet-office.gov.uk
+
+  PerfTestApplicationLoadBalancerSecurityGroupIngress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Condition: IsStaging
+    Properties:
+      # checkov:skip=CKV_AWS_260: PLAT-477 Raised to resolve
```
Comment (ethanmills) [inline template.yaml:956]: This ticket is marked as done

**Review by ethanmills — APPROVED** [review]

File: template.yaml (hunk 1 above)
Comment (CarlyG55) [reply template.yaml:303]: ah I've now done the ones that aren't the perf test stub, so I should probably update this now. Thanks!

**Review by ethanmills — APPROVED** [review]

**Pull request merged**